web/src/App.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import App from './App';
+
+
+describe('Shallow Mount Test of App', () => {
+  it('Shallow render App Component', () => {
+    const wrapper = shallow(<App/>);
```

Comment:
I think for snapshots we should do a full mount. Im sure there is some library that has a render function you can use.